strategy/calculated_peg.go

```diff
@@ -0,0 +1,29 @@
+package strategy
+
+import (
+	"github.com/joshprzybyszewski/cribbage/cards"
+	"github.com/joshprzybyszewski/cribbage/pegging"
+)
+
+func PegHighestCardNow(hand, prevPegs []cards.Card, curPeg int) (cards.Card, bool) {
```

Comment:
It would be good to add a follow-up that also calculates the highest possible considering the next cards played.

i.e. If the opponent has played a 5h, and I have `5d`, `5s`, `7h`, `10h`, which is the best to play? Well, I could get 15 for 2, or I could play a 5 for pair 2. Wouldn't my 5 be better though because the opponent could play another 5 (15 and trip for 8) and I'd play 20 for 12. Or the opponent could play 16, and I'd play 23 for 3. 🤷‍♂